homework06/src/Fibonacci.hs

```diff
@@ -0,0 +1,75 @@
+----------------------------------------------------------------------
+-- |
+--
+-- CIS 194 Spring 2013: Homework 06
+--
+----------------------------------------------------------------------
+
+module Fibonacci where
+
+----------------------------------------------------------------------
+-- Exercise 1
+----------------------------------------------------------------------
+
+fib :: Integer -> Integer
+fib = undefined
+
+fibs1 :: [Integer]
+fibs1 = undefined
+
+
+----------------------------------------------------------------------
+-- Exercise 2
+----------------------------------------------------------------------
+
+fibs2 :: [Integer]
+fibs2 = undefined
+
+
+----------------------------------------------------------------------
+-- Exercise 3
+----------------------------------------------------------------------
+
+--streamToList :: Stream a -> [a]
```

Comment:
@flandrade Is this in comments to wait for the `Stream` implementation? Is it better if we add an empty declaration?

```haskell
data Stream a
```